src/packageParser.js

```diff
@@ -18,7 +18,7 @@ function loadAndParseFile( path ) {
 		throw new ReferenceError( 'File with given path does not exist.' );
 	}
 
-	const contents = readFileSync( path );
+	const contents = readFileSync( path ).toString();
```

Comment:
There is already some code doing such a conversion: https://github.com/Comandeer/rollup-lib-bundler/blob/d267bc6cfb9963ae0b3a354950073156238e2587/tests/bundler.js#L177

Due to the consistency it should be rather:

```suggestion
	const contents = readFileSync( path, 'utf8' );
```